rust/sdk/src/client.rs

```diff
@@ -1131,16 +1131,102 @@ impl<'a> SubscribeMonotonic<'a> {
         panic!("Calling SubscribeMonotonic::with_session_id after polling.")
     }
 
-    // TODO: Figure out how this is different from the lower bound
+    // TODO: there's info missing about the difference between Subscribe and SubscribeMonotonic
```

Comment:
SubscribeMonotonic keeps track of the highest sort order (LamportTimestamp+StreamId) seen so far and ends the stream with a TimeTravel message in case the next delivered event would be older.